src/strategies/xsgt-voting-power/index.ts

```diff
@@ -0,0 +1,34 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'bluebelt';
```

Comment:
```suggestion
export const author = 'acinmavi';
```

Reply:
Thanks @ChaituVR , I run strategy test and it worked perfectly .
I update again author. Please help to review it.